engines/director/lingo/xlibs/applecdxobj.cpp

```diff
@@ -143,18 +143,16 @@ AppleCDXObject::AppleCDXObject(ObjectType ObjectType) :Object<AppleCDXObject>("A
 	_objType = ObjectType;
 	_inpoint = 0;
 	_outpoint = 0;
-	_cue = nullptr;
 
 	Common::File cuefile;
 	if (cuefile.open("disc.cue")) {
 		Common::String cuestring = cuefile.readString(0, cuefile.size());
 
-		_cue = new Common::CueSheet(cuestring.c_str());
+		_cue = Common::SharedPtr<Common::CueSheet>(new Common::CueSheet(cuestring.c_str()));
 	}
 }
 
 AppleCDXObject::~AppleCDXObject() {
```

Comment:
It should be safe to delete this entirely at this point.

Reply:
Done